.github/workflows/cifuzz.yml

```
@@ -0,0 +1,35 @@
+name: CIFuzz
+on: [pull_request]
+permissions: {}
+jobs:
+  Fuzzing:
```

A very minor detail: this project has a convention of writing job labels in all lower case, so it would be good to follow suit here. Also, it's good to fill out the `name` field to provide a more readable name in the GUIs.

```suggestion
  fuzzing:
    name: Fuzz testing
```